### PR TITLE
Embed OpenSAML2 dependencies

### PR DESCRIPTION
**Pull request by deshankoswatte**
## Purpose
- To remove exposure of OpenSAML2 dependencies to product-is

## Goals
- Embed the required OpenSAML2 dependencies.

### CODE REVIEW

File: modules/orbit/pom.xml [hunk 1]
```diff
@@ -66,6 +66,29 @@
...
+            <artifactId>opensaml1</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.opensaml</groupId>
+            <artifactId>opensaml</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.opensaml</groupId>
+            <artifactId>xmltooling</artifactId>
+            <version>1.4.6</version>
```
Comment (GDRDABARERA) [inline modules/orbit/pom.xml]: shall we  put properties or either refer by parent pom

Comment (deshankoswatte) [reply modules/orbit/pom.xml]: Moved into properties.

File: modules/orbit/pom.xml [hunk 2]
```diff
@@ -66,6 +66,29 @@
...
+            <artifactId>opensaml1</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.opensaml</groupId>
+            <artifactId>opensaml</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.opensaml</groupId>
+            <artifactId>xmltooling</artifactId>
+            <version>${opensaml.xmltooling.version}</version>
```
Comment (madurangasiriwardena) [inline modules/orbit/pom.xml]: can we add these versions to the dependency management section in the root pom of the project and get rid of them from here?

Comment (deshankoswatte) [reply modules/orbit/pom.xml]: Yeah i tried that too but when you add to the root pom then, tests are failing

Comment (deshankoswatte) [reply modules/orbit/pom.xml]: This was fixed by remove xmltooling and openws dependencies

**Pull request merged**